inst/starter_models/macpan_base/model.R

```diff
@@ -0,0 +1,52 @@
+library(macpan2)
+
+## expression lists
+###################################################
+
+## free form computations for convenience
+computations = list(
+  N ~ sum(S, E, Ia, Ip, Im, Is, R, H, ICUs, ICUd, H2, D)
```

Comment:
I don't think that `D` should be in the sum, as dead individuals should have no influence on transmission in COVID.

Reply:
As we may have discussed before, this is a gray area, and is probably only conceptually (not practically) important. Honestly, if we are modeling most human diseases, the impact of removal by death on disease dynamics is negligible; we scale `beta*S*I` by `N` to get parameters that are easier to interpret ($\cal O(1)$ rather than $\cal(O(1/N))$, not to make a point about frequency- vs density-dependent transmission. Honestly, if I were coding this model myself I would be more likely to make `N` a *parameter* and then set up "before" code that specified initial conditions as (e.g. `S ~ N-E0; E~E0; Ia ~ 0`; etc.) (which reminds me: it would be nice to have an example of non-trivial computations of initial conditions as a function of parameters [e.g., set initial values to equilibrium, or something like that])

Reply:
Thanks @bbolker.  This makes perfect sense to me.  It will be trivial to specify the model the way that you suggest, so let's do it.